htsinfer/mapping.py

```diff
@@ -299,6 +299,7 @@ def build_star_command(
                 "--runThreadN", f"{str(self.threads_star)}",
                 "--genomeDir", f"{str(index_dir)}",
                 "--outFilterMultimapNmax", "50",
+                "--outSAMorder", "PairedKeepInputOrder",
```

Comment:
But does this handle cases where the inputs, as provided by the users, are unsorted/weirdly sorted?

If not, I guess it's fine to keep it like this for now, but maybe you could document the assumption of properly sorted files for library type inference somewhere, and you could open a low priority issue for implementing sorting of the inputs so as to remove that assumption.